modules/serverless-logic-sending-rest-requests-openAPI-specification.adoc

```diff
@@ -0,0 +1,212 @@
+// Module included in the following assemblies:
+//
+
+:_mod-docs-content-type: PROCEDURE
+[id="serverless-logic-sending-rest-requests-openAPI-specification_{context}"]
+= Sending REST requests based on the OpenAPI specification
+
+To send REST requests that are based on the OpenAPI specification files, you must perform the following procedures:
+
+* Define the function references
+* Access the defined functions in the workflow states
+
+.Prerequisites
+
+* {ServerlessLogicOperatorName} is installed on your cluster.
+* You have created a {ServerlessLogicProductName} project or have access to a project with the appropriate roles and permissions to create applications and other workloads in OpenShift Container Platform.
+* You have created your {ServerlessLogicProductName} project
+* You have access to the OpenAPI specification files.
+
+.Procedure
+
+. To define the OpenAPI functions: 
+
+.. Identify and access the OpenAPI specification files for the services you intend to invoke.
+
+.. Copy the OpenAPI specification files into your workflow service directory, such as `src/main/resources/specs`.
++
+Here is an example of the OpenAPI specification for the multiplication REST service:
++
+.Example multiplication REST service OpenAPI specification
+[source,yaml]
+----
+openapi: 3.0.3
+info:
+  title: Generated API
+  version: "1.0"
+paths:
+  /:
+    post:
+      operationId: doOperation
+      parameters:
+        - in: header
+          name: notUsed
+          schema:
+            type: string
+          required: false
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/MultiplicationOperation'
+      responses:
+        "200":
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  product:
+                    format: float
+                    type: number
+components:
```

Comment:
components: segment is correclty indented?

Reply:
Yes, I verified.